secure-mobile-push/smp-android-client/securemobilepush/src/main/java/de/adorsys/android/securemobilepush/sms/SmsReceiver.java

```diff
@@ -26,30 +27,39 @@
     @Override
     public void onReceive(Context context, Intent intent) {
         if (intent.getAction().equals(INTENT_ACTION_SMS_RECEIVED)) {
-            List<String> smsSenderNumbers = Arrays.asList(BuildConfig.SMS_SENDER_NUMBERS);
+            List<String> smsSenderNumbers = Arrays.asList(SmsConfig.SMS_SENDER_NUMBERS);
             Bundle bundle = intent.getExtras();
             SmsMessage[] smsMessages;
-            String messageFrom;
+            String messageFrom = null;
             if (bundle != null) {
                 try {
                     //PDU = protocol data unit
                     //A PDU is a “protocol data unit”, which is the industry format for an SMS message.
-                    //Because SMSMessage reads/writes them you shouldn’t need to disect them.
+                    //Because SMSMessage reads/writes them you should'nt need to dissect them.
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/pull/44/commits/e66adc11b4a7a93595cda41a2e21b2e3ffd3fc85